test/sync/src/androidTest/kotlin/io/realm/kotlin/test/shared/SyncedRealmTests.kt

```diff
@@ -71,21 +77,25 @@ import kotlin.test.assertNotNull
 import kotlin.test.assertTrue
 import kotlin.test.fail
 import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.Duration.Companion.seconds
 
+@Suppress("LargeClass")
 class SyncedRealmTests {
 
     companion object {
         // Initial version of any new typed Realm (due to schema being written)
         private val INITIAL_VERSION = VersionId(2)
     }
 
+    private lateinit var tmpDir: String
```

Comment:
If not needed throughout the test maybe just have this a local variable in `createWriteCopyLocalConfig`